### PR TITLE
Python connectors

### PR DESCRIPTION
**Pull request by ldom**

### CODE REVIEW

**Review by bbejeck — COMMENTED** [review]
Thanks for the contribution @ldom - I like the tutorial.
I have a couple of comments, but otherwise this is looking good.

File: python-connectors/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+### Note on SMTs
+
+SMTs are functions that transform the messages before they are saved into the source topic by the source connector. 
+
+They receive the messages with some context and return the transformed (or not) messages. They can even return `None` to filter messages out (in other words, those won't be saved into the input topic). 
+
+In our example (no 3 above), the connector and the SMT are on the same script. This is for convenience but not mandatory at all. As a matter of fact, in the vast majority of the case, the SMTs will be fully separated from the connectors.
+
+## A brief look at the code
```
[the remark this answers is not in the document]
Comment (ldom) [reply python-connectors/README.md]: Yes you're right! I have updated the doc.

File: python-connectors/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+There are 3 examples provided with this tutorial in the `json`folder. They all use a python connector and the last one also includes a python SMT.
+
+1. `connector-fields.json`: this example publishes JSON messages containing 6 fields. This corresponds to the `code/speed_traps.py` script.
+2. `connector-full-text.json`: this example publishes messages with one big string as the value. This is in the `code/full_text.py` script.
+3. `connector-with-smt.json`: this example publishes simple messages and the same script also contains an SMT that is modifying the messages. It can be found in the `code/connector_with_smt.py` script.
+
+To create a connector you can use `curl` or you can use the Control Center UI. Since we already have a terminal open, let's use that.
+
+Run `curl localhost:8083/connectors -X POST --json @/json/connector-fields.json` to create a connector with the example 1 above.
```
[the remark this answers is not in the document]
Comment (ldom) [reply python-connectors/README.md]: I have corrected the link but there's an issue with the jar generated by semaphore. I'm sending a proper one to a different location while I resolve this other issue. (Will update this thread when done.)

[the remark this answers is not in the document]
Comment (ldom) [reply python-connectors/README.md]: this should be good now, thanks

**Pull request closed without merging**